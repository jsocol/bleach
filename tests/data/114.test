<div id="42"><head><base href="javascript://"/></head><body><a href="/. /,alert(42)//#">XXX</a></body>//["'`-->]]>]</div>
