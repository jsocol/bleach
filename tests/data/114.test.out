<div id="42"><a href="/. /,alert(42)//#">XXX</a>//["'`--&gt;]]&gt;]</div>
